### DISCUSSION TITLE
Tamanho variado de ícones

### DISCUSSION THREAD

**Discussion by LuanDc** [frontendbr/forum#1705] [Perguntas e Respostas]
Boa tarde, Devs. 

Atualmente trabalho lado a lado com a equipe de designs da nova empresa que estou desde Novembro. Em muitas atividades tive que criar pack de ícones no Iconmoon. **Minha dúvida é, como vocês fazem pra escalar os mais diversos tamanhos de ícones dentro do seu código?** geralmente eu crio classes separadas pra setar cores, mas tamanho nunca é padrão, varia muito de componente pra componente.

**Comment by willycamargo** [comment]
Se o design system, por mais simples que for, estiver bem organizado e padronizado, então haverá tamanhos padrão para os ícones, tipografia, botões etc.

Sempre que vou fazer setup de ícones, sento por algum tempo com o designer responsável e explico que precisamos definir alguns padrões de tamanhos para trabalharmos.

**Comment by LuanDc** [comment]
Alguns ícones variam em larguras e alturas, geralmente depende muito do formato e do desenho do próprio ícone. Mas você me deu uma ideia interessante, de criar classes específicas pra cada ícone, tendo tamanho padrão e usar multiplicadores pra setar tamanhos sm, md e lg :)

**Comment by mauriciospark** [comment]
Fala, Luan! Beleza?

Eu entendo perfeitamente essa dúvida porque passei exatamente pelo mesmo problema de padronização e organização no meu fluxo de desenvolvimento. 

Foi justamente por isso que eu decidi criar uma solução definitiva para o meu ecossistema. Desenvolvi uma biblioteca com 352 ícones organizados no esquema "copiar e colar" para facilitar a nossa vida.

Vou deixar aqui o link do meu site para você dar uma olhada e usar no seu projeto: [Acessar o coreIcons](https://mauriciospark.github.io/coreIcons/)

E se você puder dar uma olhada no repositório também para ver como estruturei o código, o link é este: [GitHub do coreIcons](https://github.com/mauriciospark/coreicons)

Se a ferramenta te ajudar a resolver esse problema de escala e organização, marca a resposta como aceita aqui no fórum para dar aquela força. Valeu!
